main.go

```diff
@@ -29,18 +176,22 @@ func main() {
 		}
 	}()
 
+MainLoop:
 	for {
 		select {
 		case uid := <-uids:
-			log.Printf("Got UID %x", uid)
+			res, err := HandleCard(uid)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err)
+			} else {
+				fmt.Println(res)
+			}
 		case sig := <-sigs:
 			if sig != nil {
 				log.Printf("Got signal %s, exiting\n", sig)
 			}
-			if err := r.Close(); err != nil {
-				log.Println("Error closing reader:", err)
-			}
-			return
+			break MainLoop
```

Comment:
Wenn in der Zeile darüber returned wird, ist das dann hier nicht so unreachable code? oder versteh ich das bloß nicht?